index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <img src="images for portfolio/logo.jpg" alt="" class="logo">
+                <a class="logo-text">Sameed Mohsin</a>
+            </div>
+
+            <div class="menu">
+                <img src="./Images for portfolio/Hamburger.svg" alt="" class="hamburger" width="50px" height="50p">
+            </div>
```

Comment:
To keep the structure and style separate, don't use Inline Styling. Put an `id` to this element, and add its CSS in `style.css` file. This will also help in the organization of the project when you will enlarge this project.